src/test/java/no/nav/pto/veilarbportefolje/database/BrukerRepositoryTest.java

```diff
@@ -91,12 +91,6 @@ public void skal_hente_bruker_fra_view() {
         assertThat(bruker).isPresent();
     }
 
-    @Test
-    public void skal_returnere_riktig_antall_brukere_under_oppfolging() {
-        List<OppfolgingsBruker> brukereUnderOppfolging = brukerRepository.hentAlleBrukereUnderOppfolging(0, ANTALL_LINJER_I_TESTDATA);
-        assertThat(brukereUnderOppfolging.size()).isEqualTo(ANTALL_OPPFOLGINGSBRUKERE_I_TESTDATA);
-    }
-
```

Comment:
Er ikke dette en ganske kritisk test å fjerne? Den er kanskje litt for banal?

Reply:
Fjernet metoden den tester, for den brukes bare ved hovedindeksering som nå ligger i `veilarbindexer` :)

Reply:
Riktig. De største tingene som ble fjernet var vel rester fra veilarbindexer? Tenker at PRen ser grei ut for min del i alle fall 👍

Reply:
Trykker du på godkjenn i så fall? 😛